python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py

```diff
@@ -0,0 +1,368 @@
+from __future__ import print_function
+
+import datetime
+from collections import namedtuple
+
+import os
+import random
+import tempfile
+import time
+from textwrap import dedent
+
+import pytest
+
+from Ganga.Utility.logging import getLogger
+from GangaDirac.Lib.Utilities.DiracUtilities import execute
+
+from Ganga.testlib.mark import external
+from Ganga.testlib.GangaUnitTest import load_config_files, clear_config
+
+logger = getLogger(modulename=True)
+
+random.seed()
```

Comment:
I don't know/remember I think I added some quick and dirty code for making random file contents a while back and it seems to have been copied verbatim. I'll replace this with an appropriate uuid call.